code-challenges/401/arrayReverse/array-reverse.js

```diff
@@ -0,0 +1,18 @@
+// Write a function that takes an array as an argument. Without utilizing any of the methods in my language, return an array with elements in reverse order.
+
+const arr = [1, 2, 3, 4, 5, 6];
+
+const initial = 0;
+const lastElement = arr.length -1;
+
+while (inital < lastElement)
+  {var x = arr[i];
```

Comment:
initial is spelled incorrectly

Reply:
Good catch! I have noticed that when I work with a migraine or tension headache spelling goes down the drain. I will stop working when I am in that state in the future. Thank you, again, for your feedback!